fuzzers/int_loop.sh

```diff
@@ -43,6 +43,10 @@ i=1
 while true; do
     ${MAKE} ITER=$i cleaniter
     ${MAKE} ITER=$i build/todo.txt
+    if [ ! -s build/todo.txt ]; then
```

Comment:
I don't think this helps, because the problem is that pushdb fails because there is nothing to push.

Reply:
Actually it helps in a case where empty todo was generated in the first iteration. Currently if such a situation happens the code breaks the loop and goes directly to pushdb and fails on empty segbits.

I'll extend the if empty todo with first iteration check.

Reply:
done, now the check is performed only for the first loop iteration